app/code/Magento/ConfigurableProductGraphQl/Model/Variant/Collection.php

```diff
@@ -160,4 +164,24 @@ private function fetch() : array
 
         return $this->childrenMap;
     }
+
+    /**
+     * Get attributes code
+     *
+     * @param \Magento\Catalog\Model\Product $currentProduct
+     * @return array
+     */
+    private function getAttributesCode(Product $currentProduct): array
```

Comment:
Looks like we can use more simple way for loading all of the possible (visible) attribute codes instead of loading attributes models